sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatLiveTestBase.cs

```diff
@@ -6,15 +6,16 @@
 using System.Threading.Tasks;
 using Azure.Communication.Administration;
 using Azure.Communication.Identity;
-using Azure.Communication.Pipeline;
 using Azure.Core.TestFramework;
 
 namespace Azure.Communication.Chat.Tests
 {
     public class ChatLiveTestBase : RecordedTestBase<ChatTestEnvironment>
     {
         public ChatLiveTestBase(bool isAsync) : base(isAsync)
-           => Sanitizer = new ChatRecordedTestSanitizer();
```

Comment:
this was changed to expression body format to look like other constructors we have, any reason why we are changing it back?

Reply:
done